README.md

```diff
@@ -65,6 +65,10 @@ Generate pdf file with pkcs#1 schema. This action will create a pdf file name `s
 ```bash
 yarn pdf:pcd
 ```
+You can generate more pdf files have a same certificate by command: 
```

Comment:
You can generate several PDF files with the same certificate by running the following command:
```bash
yarn pdf:pcd other-file.pdf #other-file can be any name you want to give to the file
```